templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul class="sidebar-nav">
+  <li><a href="{% url 'taxi:index' %}">Home page</a></li>
+  <li><a href=#>Manufacturers</a></li>
+  <li><a href=#>Cars</a></li>
+  <li><a href=#>Drivers</a></li>

```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are currently placeholders (`href=#`). Once the corresponding views and URL patterns are set up, replace `#` with the appropriate `{% url %}` template tags to ensure proper navigation.